viewing_party/party.py

```diff
@@ -1,23 +1,225 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+
+    # those three attributes are truthy
+    if not (title and genre and rating):
+        return None
+    
+    if title and genre and rating:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating

```

Comment:
If we have a check previous to this that checks for `Falsyness`, then do you we need a check for `Truthyness`? They are inherently checking the same thing since a boolean can only either `True` or `False`. 
```suggestion
    movie["title"] = title
    movie["genre"] = genre
    movie["rating"] = rating
```